learn/tools-ides/vscode-plugin/run-and-debug.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: ballerina-inner-page
 title: Run and debug
-permalink: /learn/vscode-plugin/run-and-debug
+permalink: /learn/vscode-plugin/run-and-debug/
```

Comment:
don't we have to add a redirection_from to `/learn/vscode-plugin/run-and-debug` ... without the forward slash in the end?